README.md

```diff
@@ -104,7 +104,7 @@ If you built the images (or if you already have the images locally from a previo
 
 After starting the demo with the defaults, the following services should work.
 
-  - Ember application: [https://localhost](https://localhost)
+  - Ember application: [https://localhost](https://localhost). _See [Shibboleth users](#shibboleth-users) below for login options_ 
```

Comment:
Oh interesting; I've been using `https://pass`.  I'm not sure what the consequences are of using `https://localhost` might be.  I think some links (from the indexer, and maybe user service) will have `https://pass` in them; but everything might work just fine, as those are definitely resolvable.  To a developer, it may look a little weird though.